src/components/CustomerQuota/ItemsSelection/ItemsSelectionCard.tsx

```diff
@@ -149,17 +126,10 @@ export const ItemsSelectionCard: FunctionComponent<ItemsSelectionCard> = ({
               !isChargeable
                 ? checkoutCart
                 : () => {
-                    showAlert({
-                      ...defaultConfirmationProps,
-                      title: "Payment collected?",
-                      description: WARNING_MESSAGE.PAYMENT_COLLECTION,
-                      buttonTexts: {
-                        primaryActionText: "Collected",
-                        secondaryActionText: "No"
-                      },
-                      visible: true,
-                      onOk: checkoutCart
-                    });
+                    showConfirmationAlert(
+                      new Error(WARNING_MESSAGE.PAYMENT_COLLECTION),
+                      checkoutCart
+                    );
```

Comment:
Same thing for this, this is not really an error, so maybe we can let `showConfirmationAlert` take in an enum rather than an error?

```
showConfirmationAlert(WARNING_MESSAGE.PAYMENT_COLLECTION, checkoutCart);
```

Reply:
This has been refactored to:
```suggestion
                    showConfirmationAlert(
                      CONFIRMATION_MESSAGE.PAYMENT_COLLECTION,
                      checkoutCart
                    );
```